src/Gateways/GpApiGateway.php

```diff
@@ -550,6 +567,40 @@ public function handle_adding_capture_order_action( $actions ) {
 		return $actions;
 	}
 
+	/**
+	 * Handles subscription renewals, gets the muti-use token from the order meta and charges it. 
+	 * @return bool
+	 */
+
+	public function renew_subscription($amount_to_charge, $renewal_order)
+	{
```

Comment:
Braces should be on the same line with the function/class declaration  -https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#brace-style